src/neighbor.cpp

```diff
@@ -2956,4 +2948,4 @@ double Neighbor::memory_usage()
   if (neigh_improper) bytes += neigh_improper->memory_usage();
 
   return bytes;
-}
+}
```

Comment:
Please make certain that there is a newline character at the end of the file (check your editor settings, if necessary).